ResourceStatusSystem/Client/SiteStatus.py

```diff
@@ -43,7 +43,6 @@ def __init__( self ):
     self.__opHelper = Operations()
     self.rssFlag = ResourceStatus().rssFlag
     self.rsClient = ResourceStatusClient()
-    self.wmsAdministrator = RPCClient( 'WorkloadManagement/WMSAdministrator' )
```

Comment:
Depending from the client that calls it, the RPCClient object may not be yet fully initialized.